### PR TITLE
Dpetry fix

### PR DESCRIPTION
**Pull request by danpetry**
{package} {version} {:snowflake:}

**Destination channel:** {Snowflake | defaults}

### Links

- [{ticket_number}]() 
- [Upstream repository]()
- [Upstream changelog/diff]()
- Relevant dependency PRs:
  - ...

### Explanation of changes:

- ...

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+build:
+  number: 0
+  skip: True  # [py>311]
+  script:
+    - export USE_FFMPEG=ON          # [not win]
+    - set USE_FFMPEG=ON             # [win]
+    - export BUILD_SOX=OFF          # [not win]
+    - set BUILD_SOX=OFF             # [win]
+    - export FFMPEG_ROOT=${PREFIX}  # [not win]
+    - set FFMPEG_ROOT=%PREFIX%      # [win]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
    - set FFMPEG_ROOT=%LIBRARY_PREFIX%      # [win]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    - llvm-openmp  # [osx]
+    - intel-openmp  # [not osx and blas_impl == 'mkl']
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+    - pytorch 2.1.0
+    - pybind11
+    - libabseil
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: From what I saw in my local experiments, `libabseil` is not required or even used.

**Comment by danpetry** [comment]
The tests fail on osx-arm64 with:
```
FAILED test/integration_tests/ctc_decoder_integration_test.py::test_decoder_from_pretrained[librispeech-expected0] - RuntimeError: CTC Decoder suit requires flashlight-text package and optionally KenLM. Please install them.
FAILED test/integration_tests/ctc_decoder_integration_test.py::test_decoder_from_pretrained[librispeech-3-gram-expected1] - RuntimeError: CTC Decoder suit requires flashlight-text package and optionally KenLM. Please install them.
FAILED test/integration_tests/tacotron2_pipeline_test.py::test_tts_models[bundle1] - RuntimeError: DeepPhonemizer is not installed. Please install it.
FAILED test/integration_tests/tacotron2_pipeline_test.py::test_tts_models[bundle3] - RuntimeError: DeepPhonemizer is not installed. Please install it.
FAILED test/integration_tests/wav2vec2_pipeline_test.py::test_finetune_asr_model[bundle8-en-I|HAVE|THAT|CURIOSITY|BESIDE|ME|AT|THIS|MOMENT|] - RuntimeError: PytorchStreamReader failed reading zip archive: failed finding central directory
```
these can probably be skipped as they're due to missing resources

File: recipe/meta.yaml (hunk 2 above)
Comment (danpetry) [reply recipe/meta.yaml]: removed

**Comment by boldorider4** [comment]
closing this PR because I cherry picked all commits to main PR.

**Pull request closed without merging**